packages/calcite-components/src/components/swatch/swatch.stories.ts

```diff
@@ -0,0 +1,63 @@
+import { boolean, modesDarkDefault } from "../../../.storybook/utils";
+import { html } from "../../../support/formatting";
+import { ATTRIBUTES } from "../../../.storybook/resources";
+import { Swatch } from "./swatch";
+
+const { scale } = ATTRIBUTES;
+
+type SwatchStoryArgs = Pick<Swatch, "scale" | "selected" | "label">;
+
+export default {
+  title: "Components/Swatch",
+  args: {
+    scale: scale.defaultValue,
+    selected: false,
+    label: "My great swatch",
+  },
+  argTypes: {
+    scale: {
+      options: scale.values,
+      control: { type: "select" },
+    },
+    label: {
+      control: { type: "text" },
+    },
+  },
+};
+
+export const simple = (args: SwatchStoryArgs): string => html`
+  <div style="background-color:white;padding:100px">
+    <calcite-swatch scale="${args.scale}" label="${args.label}" ${boolean("selected", args.selected)}></calcite-swatch>
+  </div>
+`;
+
+export const withIcon = (args: SwatchStoryArgs): string => html`
+  <div style="background-color:white;padding:100px">
+    <calcite-swatch scale="m" label="${args.label}"> </calcite-swatch>
+  </div>
+`;
+
+export const withImage = (args: SwatchStoryArgs): string => html`
```

Comment:
Not seeing an image in storybook

Reply:
Updated stories, will update image paths subsequently.